agriculture/agriculturecore/views.py

```diff
@@ -283,10 +287,13 @@ def get_irrigation_stations(request):
             Stations corresponding to the controller with the ID provided in
             the request.
     """
-    # Check if the AJAX request is valid.
-    error = check_ajax_request(request)
```

Comment:
This method does the same as the code you have placed below. Why was the call removed and the code replicated? If you want to change the way the method works, you can always create a new method and reuse it everywhere, that way you just write the code once and reuse everywhere.